color_windows.go

```diff
@@ -11,8 +11,10 @@ func init() {
 	// https://learn.microsoft.com/en-us/windows/console/console-virtual-terminal-sequences#output-sequences
 	var outMode uint32
 	out := windows.Handle(os.Stdout.Fd())
-	if err := windows.GetConsoleMode(out, &outMode); err == nil {
-		outMode |= windows.ENABLE_PROCESSED_OUTPUT | windows.ENABLE_VIRTUAL_TERMINAL_PROCESSING
-		_ = windows.SetConsoleMode(out, outMode)
+	if err := windows.GetConsoleMode(out, &outMode); err == nil || NoColor {
```

Comment:
```suggestion
	if err := windows.GetConsoleMode(out, &outMode); err == nil {
```

see https://github.com/fatih/color/pull/186#discussion_r1121285897

Reply:
I guess it would be better to add opt-out code as a separate PR 😉